src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -2407,17 +2409,24 @@ long SurgeGUIEditor::applyParameterOffset(long id)
 void SurgeGUIEditor::setZoomFactor(int zf)
 {
 #if HOST_SUPPORTS_ZOOM    
+
+#if TARGET_VST2 || TARGET_VST3
+   // vstgui zoom is very unreliable shrinking with backgrounds.
+   if( zf < 100 )
+       zf = 100;
+#endif
```

Comment:
Not sure if I understand what the comment is trying to say about what you are doing here.